openpype/pipeline/farm/pyblish_functions.py

```diff
@@ -568,9 +568,15 @@ def _create_instances_for_aov(instance, skeleton, aov_filter, additional_data,
             col = list(cols[0])
 
         # create subset name `familyTaskSubset_AOV`
-        group_name = 'render{}{}{}{}'.format(
-            task[0].upper(), task[1:],
-            subset[0].upper(), subset[1:])
+        # TODO reactor/remove me
+        family = skeleton["family"]
+        if not subset.startswith(family):
+            group_name = '{}{}{}'.format(
+                family,
+                task.capitalize(),
+                task.capitalize())
```

Comment:
```suggestion
                subset.capitalize())
```

Reply:
Pretty sure capitalize is NOT what you want. This is introducing bugs or at least very different behavior than before.

```python
"myStringVlue Hello".capitalize()
# Result: Mystringvlue hello # 
```

Instead of only uppercasing the first character this now ALSO lowercases all other characters.

Reply:
Pwned, so this is why there was that weird construct before, reverted back.